test/api/todos.coffee

```diff
@@ -97,6 +97,36 @@ describe "Todos", ->
           expect(todo.value).to.equal 0
           done()
 
+      it "Creates a completed a todo when using up url", (done) ->
+        upTodo = undefined
+        request.post(baseURL + "/user/tasks/withUp/up").send(
+            type: "todo"
+            text: "testTitle"
+            complete: true
```

Comment:
I think you should not pass this parameter if you're checking that the scoring route does the work of completing or uncompleting it.